conf/nginx.conf

```diff
@@ -0,0 +1,21 @@
+
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+  server {
+    include    mime.types;
+
+    listen 80 ;
+    server_name app.mydomain.com;
+    root /usr/share/nginx/html;
+    rewrite (\/web_dev)(.*) $2 last;
```

Comment:
what is the idea behing that `rewrite` ?

Reply:
because we don't have folder /web_dev but files are compiled with /web_dev in URL so the rewrite is removing it.